docs/tutorials/basic/go.md

```diff
@@ -510,6 +510,7 @@ if err != nil {
 }
 for _, point := range points {
 	if err := stream.Send(point); err != nil {
+		_, err := stream.CloseAndRecv()
```

Comment:
How about `break` if err is `io.EOF`, so `CloseAndRecv` after this for loop will get the error?

Reply:
sure that makes it more in-line with the server streaming example, updated